chef_master/source/ruby.rst

```diff
@@ -612,11 +618,13 @@ Constructs to Avoid
 -----------------------------------------------------
 Avoid the following patterns:
 
+* ``node.normal` - Avoid using attributes at normal precedence since they are set directly on the node object itself, rather than implied (computed) at runtime.
```

Comment:
```suggestion
* ``node.normal`` - Avoid using attributes at normal precedence since they are set directly on the node object itself, rather than implied (computed) at runtime.
```